force-app/main/default/classes/OpportunitySummaryByEmailBatch.cls

```diff
@@ -0,0 +1,66 @@
+@isTest
+public class OpportunitySummaryByEmailBatchTest {
```

Comment:
This appears to be the test class, contained in the OpportunitySummaryByEmailBatch.cls  
Please add the OpportunitySummaryByEmailBatchTest  class in OpportunitySummaryByEmailBatchTest  file and  commit also the OpportunitySummaryByEmailBatch.cls